misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenNodeFactory.java

```diff
@@ -909,58 +924,83 @@ private static StatementNode createReturnIfHandleIsNullStatement(final String ha
+                    SEMICOLON_TOKEN
+            );
+        } else {
+            // builds the code fragment 'return [];'
+            returnStatement = NodeFactory.createReturnStatementNode(
+                    createToken(
+                            SyntaxKind.RETURN_KEYWORD,
+                            EMPTY_WHITE_SPACE,       // no leading whitespace
+                            SINGLE_SPACE_WHITE_SPACE // one blank as trailing whitespace
+                    ),
+                    ARRAY_TOKEN_EXPRESSION,
+                    SEMICOLON_TOKEN
+            );
+        }
 
         // builds the code fragment
         //   if java:isNull(<handleName>) {
-        //       return null;
+        //       return <() / []>;
         //   }
         return NodeFactory.createIfElseStatementNode(
-            NodeFactory.createToken(
-                SyntaxKind.IF_KEYWORD,
-                EMPTY_WHITE_SPACE,       // no leading whitespace
-                SINGLE_SPACE_WHITE_SPACE // one blank as trailing whitespace
-            ),
-            // the expression
-            checkIsNullExpression,
-            // the if block
-            createBlockStatementNode(
-                NodeFactory.createNodeList(returnStatement)
-            ),
-            null // no else block
+                createToken(
+                        SyntaxKind.IF_KEYWORD,
+                        EMPTY_WHITE_SPACE,       // no leading whitespace
+                        SINGLE_SPACE_WHITE_SPACE // one blank as trailing whitespace
+                ),
+                // the expression
+                checkIsNullExpression,
+                // the if block
+                createBlockStatementNode(NodeFactory.createNodeList(returnStatement)),
+                null // no else block
         );
     }
 
     @SuppressWarnings("SpellCheckingInspection")
     private static List<StatementNode> getStringArrayReturnStatements(JMethod jMethod) {
         return List.of(
-            //   handle externalObj = <call external method>;
-            getExternalFunctionCallStatement(HANDLE, jMethod),
-            //   if java:isNull(externalObj) {
-            //       return null;
-            //   }
-            createReturnIfHandleIsNullStatement("externalObj"),
-            //   return <string[]> check jarrays:fromHandle(externalObj, "string");
-            createReturnStatementNode(createTypeCastExpressionNode("string[]",
-                createCheckExpressionNode(
-                    createFunctionCallExpressionNode(
-                        "jarrays:fromHandle",
-                        new LinkedList<>(Arrays.asList("externalObj", "\"string\""))
-                    )
-                )
-            ))
+                //   handle externalObj = <call external method>;
+                getExternalFunctionCallStatement(HANDLE, jMethod),
+                //   if java:isNull(externalObj) {
+                //       return null;
+                //   }
```

Comment:
Shall we add this comment to java block comments?  and for other related places as well.

Reply:
Since the same comment style has been used within the entire bindgen code base, shall we keep it as it is for now?

Reply:
Yeah sure +1 for suggestions.